KSystemInformer/verify.c

```diff
@@ -356,7 +356,6 @@ NTSTATUS KphVerifyFile(
  * strict and lacks enough information from the kernel to fully understand the
  * protected process state.
  *
- * \param[in] Client - Client information.
```

Comment:
👍 - I'm also now noticing that the masks for protection are misspelled too

```diff
- KPH_PROTECED_PROCESS_MASK
+ KPH_PROTECTED_PROCESS_MASK
```

Reply:
I went ahead and removed this in this commit https://github.com/winsiderss/systeminformer/commit/5f0450fc5c876bca40a6181b42850bbfb86eb71c. I had another misc cleanup change inside `informer_debug.c`, and this was a good excuse to lump that change into it.